File: python-misinformation/secure7.py
from openai import OpenAI
import logging

client = OpenAI()
logger = logging.getLogger(__name__)

def enrich_document(doc_text: str) -> dict:
    response = client.chat.completions.create(
        model="gpt-4o",
        max_tokens=500,
        messages=[{"role": "user", "content": f"Extract key entities from:\n\n{doc_text}"}],
    )
    entities = response.choices[0].message.content
    logger.info("entities extracted for document")
    return {"entities": entities, "original": doc_text}

File: javascript-sensitiveinfodisclosure/secure7.js
const { OpenAI } = require("openai");

const client = new OpenAI({ apiKey: process.env.OPENAI_API_KEY });

async function handleUserQuery(req) {
  const userQuestion = req.body.question;
  const response = await client.chat.completions.create({
    model: "gpt-4o",
    max_tokens: 1024,
    messages: [
      { role: "user", content: userQuestion }
    ],
  });
  return "Note: AI-generated content. Verify independently.\n\n" + response.choices[0].message.content;
}
